rest/config_startup.go

```diff
@@ -138,9 +143,11 @@ type AuthConfig struct {
 }
 
 type ReplicatorConfig struct {
-	MaxHeartbeat              *base.ConfigDuration `json:"max_heartbeat,omitempty"    help:"Max heartbeat value for _changes request"`
-	BLIPCompression           *int                 `json:"blip_compression,omitempty" help:"BLIP data compression level (0-9)"`
-	MaxConcurrentReplications int                  `json:"max_concurrent_replications,omitempty" help:"Maximum number of replication connections to the node"`
+	MaxHeartbeat                *base.ConfigDuration `json:"max_heartbeat,omitempty"    help:"Max heartbeat value for _changes request"`
+	BLIPCompression             *int                 `json:"blip_compression,omitempty" help:"BLIP data compression level (0-9)"`
+	MaxConcurrentReplications   int                  `json:"max_concurrent_replications,omitempty"    help:"Maximum number of replication connections to the node"`
+	MaxConcurrentChangesBatches int                  `json:"max_concurrent_changes_batches,omitempty" help:"Maximum number of changes batches to process concurrently per replication (1-5)"`
```

Comment:
It would be nice to extend a test in `config_startup_test.go` just in case we refactor parsing.

And get a test for the config validation in `server_context.go`.

Is there a reason to not make this `*int` to represent that this is optional? Should we state the default in the help, or that this is optional?

Reply:
For the test, I think I commented on the wrong line, but I was thinking to make sure that command line flags `replicator.max_concurrent_changes_batches` etc will make it the startup config.